### ISSUE TITLE
[BUG] QB-MechanicJob Optimization

### ISSUE THREAD

**Issue by Tamer173** [qbcore-framework/qb-mechanicjob#60]
**Describe the bug**

QB-MechanicJob Optimization
I recently realised that qb-mechanicjob by default has a seemingly currently unnecessary database change, every 2 seconds of driving a vehicle.
Some servers have over 100 people in. If all players are in cars at the same time that would be 100 database changes every 2 seconds.
Long explanation is that the distance travelled is set to 0 on script/server starts
As you drive your car it updates the distance travelled.
This is then used to calculate damages and other things.
This is saved server side but also updates the database with the numbers.
This isn't needed to be saved as it is saved server side already and never calls it FROM the database and is just a waste and laggy
TLDR: qb-mechanicjob has a useless database check that can cause lag for servers with lots of players

**-- Go to qb-mechanicjob server/main.lua and find:**
`RegisterNetEvent('qb-vehicletuning:server:UpdateDrivingDistance', function(amount, plate)`

**--or go to line no 126 and**

-- Remove or comment out:
```
local result = MySQL.Sync.fetchAll('SELECT plate FROM player_vehicles WHERE plate = ?', {plate})
if result[1] ~= nil then
    MySQL.Async.execute('UPDATE player_vehicles SET drivingdistance = ? WHERE plate = ?', {amount, plate})
end
```

**Comment by nzkfc** [comment]
Nice find, might play with this and see if it improves our performance as we have 180 people on daily

**Comment by ChristianBDev** [comment]
@Tamer173 Can you make a PR for this?

**Comment by Tamer173** [comment]
> @Tamer173 Can you make a PR for this?

Whats PR?

**Comment by ChristianBDev** [comment]
> > @Tamer173 Can you make a PR for this?
> 
> Whats PR?


Pull Request, but I have already fixed it #64

**Comment by Tamer173** [comment]
> > > @Tamer173 Can you make a PR for this?
> > 
> > 
> > Whats PR?
> 
> Pull Request, but I have already fixed it #64

Ok

**Comment by Tamer173** [comment]
Fixed
